lib/util/__tests__/utimes.test.js

```diff
@@ -33,7 +33,7 @@ describe('utimes', () => {
     fse.emptyDir(TEST_DIR, done)
     // reset stubs
     gracefulFsStub = {}
-    utimes = proxyquire('../utimes', { 'graceful-fs': gracefulFsStub })
+    utimes = proxyquire('../utimes', { '../fs': gracefulFsStub })
```

Comment:
The mock here has to be replaced, as `lib/util/utimes.js` no longer uses `graceful-fs` directly. It now uses the unified API from `lib/fs/index.js`.